### ISSUE TITLE
Payments task - update eWay logo

### ISSUE THREAD

**Issue by pmcpinto** [woocommerce/woocommerce-admin#7359]
**Description:** eWay updated their brand and we need to change the logo we're displaying in the Payments task

**Acceptance criteria:**

- [ ] Replace the logo with the one available [here](https://brandfolder.com/globalpayments-aunz/eway-partner-media-kit?guest_pw=7b170fa7ef6a38076ad243fd6dbb9845377a9d0affd0c03ba13cd5dd74054570?utm_source=mkto&utm_medium=email&utm_campaign=EM%202021-06%20Register%20Your%20Interest%20Campaign&mkt_tok=MzA0LUhQQS02MjIAAAF9_2c9oRylMnJDtnnIz-sOD2XxBU26Uzum-slMuK7eVDjygzKMjWGnrUlJszioGVcdI2lpITCKt_YOA3Konfx5ayMa-FdkM-Ajx2SySuwJsg). Let's use Eway Logo RGB (Web) - Yellow

**Comment by pmcpinto** [comment]
@joshuatf this can be replaced after WC 5.6 is shipped with the remote payments task, right?

**Comment by joshuatf** [comment]
Correct, this can be modified from the WCCOM side at any point in time.

**Comment by pmcpinto** [comment]
@becdetat is it possible to implement this change in the upcoming days? The eWay extension was already updated with the new logo. Thanks!

**Comment by becdetat** [comment]
@pmcpinto I'll put this on Mothra's radar, I don't see a problem with getting it done this week
